esmvaltool/diag_scripts/bock20jgr/corr_pattern.ncl

```diff
@@ -0,0 +1,151 @@
+; #############################################################################
+; DIAGNOSTIC SCRIPT for reproducing IPCC ch. 9 fig. 9.6
+; Author: Bettina Gier (Uni Bremen & DLR, Germany)
+; CRESCENDO project
+; #############################################################################
+;
+; Description
+;    Calculates centred pattern correlations for annual mean climatologies
+;    and plots them. Like IPCC ch. 9 fig 9.6
+;
+; Required diag_script_info attributes (diagnostics specific)
+;
+; Optional diag_script_info attributes (diagnostic specific)
+;
+; Required variable_info attributes (variable specific)
+;     none
+;
+; Optional variable_info attributes (variable specific)
+;     none
+;
+; Caveats
+;
+;
+; Modification history
+;    20190205-A_gier_be: Adapted to new ncl structure
```

Comment:
Since this is newly added to the public ESMValTool, you may want to add some information to the modification history

Reply:
Thanks, I adjusted the header.